science/orthanc: Revive: DICOM server for healthcare and medical research

yzgyyang opened the pull request:
PR:		242546
Approved by:	lwhsu (mentor)

lwhsu commented on science/orthanc/Makefile:
```diff
@@ -0,0 +1,57 @@
+DISTNAME=	Orthanc-${PORTVERSION}
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Orthanc is a DICOM server for healthcare and medical research
```
I think we can drop "Orthanc is a" part.

lwhsu replied:
The old COMMENT was `RESTful DICOM server for healthcare and medical research`
https://svnweb.freebsd.org/ports/head/science/orthanc/Makefile?revision=471770&view=markup&pathrev=471893

lwhsu commented:
Don't forget encourage submitter to update the link at https://www.orthanc-server.com/download.php

lwhsu commented on science/orthanc/Makefile:
```diff
@@ -0,0 +1,57 @@
+# Created by: maintainer.freebsd@xpoundit.com
```
I somehow don't know if we should keep this created by line before it was deleted

https://svnweb.freebsd.org/ports/head/science/orthanc/Makefile?revision=471770&view=markup&pathrev=471893#l1

Or this is also fine.

I'm OK for both, maybe the new maintainer revives this is better

yzgyyang replied:
I have no strong feelings, but slightly prefer the current maintainer, so I will allow it ;)

lwhsu commented on science/orthanc/files/orthanc.in:
```diff
@@ -0,0 +1,40 @@
+
+name=orthanc
+rcvar=orthanc_enable
+desc="RESTful DICOM server for healthcare and medical research"
```
Should we align with the COMMENT in Makefile?

lwhsu commented:
> Don't forget encourage submitter to update the link at https://www.orthanc-server.com/download.php

And try to upstream the local patches.

Commit:
science/orthanc: Revive port

Orthanc is a Lightweight DICOM server for healthcare and medical
research.

PR:		242546
Approved by:	lwhsu (mentor)

lwhsu commented on science/orthanc/Makefile:
```diff
@@ -0,0 +1,57 @@
+USE_RC_SUBR=	orthanc
+
+USERS=		orthanc
+GROUPS=		orthanc
```
Note that when defining this, we also need to update ports/{UIDs,GIDs}, but it seems got forgot when deleted. :)

yzgyyang replied:
Oh! Today I learned. Though from what I can see, the UID/GID stayed the same, so it actually works as-is this time, correct?

lwhsu replied:
Yes, just letting you know this patch doesn't contain UIDs/GIDs is because of a previous fault. :)
